packages/constellation/src/components/Compositions/Headers/Settings Header/SettingsHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import cx from 'classnames'
+import React, { forwardRef } from 'react'
+
+import { Text } from '../../../Base'
+import { Button, buttonSizeStyles, rootButtonStyles } from '../../../Primitives'
+import { SectionHeader, SectionHeaderComponent } from '../Section Header'
+import { Component as ComponentType, Props } from './SettingsHeader.types'
+
+/**
+ * SettingsHeaders description
+ *
+ * When a `ref` prop is provided, it will be forwarded to the root element. Any
+ * other properties supplied will be provided to the root element (ie, the `as`
+ * prop value). This includes all appropriate HTML attributes or aria tags. This
+ * component may have its underlying DOM customized via the 'as' prop.
+ */
+
+const SettingsHeader: ComponentType = forwardRef(
+  <T extends SectionHeaderComponent>(
+    { mode = 'initial', onCancel, onEditClick, onSave, title = 'Account', ...otherProps }: Props<T>,
+    ref?: PolymorphicRef<T>,
+  ) => {
+    return (
+      <SectionHeader ref={ref} {...otherProps}>
+        <Text variant='title2' className='mt-0 mb-0 text-xl md:text-2xl'>
+          {title}
+        </Text>
+        {mode === 'initial' && (
+          <Button
+            className={cx(
+              rootButtonStyles,
+              buttonSizeStyles.small,
+              'md:px-6 md:py-3 md:text-base md:h-12',
+            )}
+            onClick={onEditClick}
+            text='Edit'
```

Comment:
Not sure is language support is needed here or not but if so the text here should probably be a prop of the component so it can be updated app side.

Reply:
Ok updated 👍